pkg/txm/clientwrappers/dualbroadcast/meta_client.go

```diff
@@ -142,7 +145,7 @@ func NewMetaClient(lggr logger.Logger, c MetaClientRPC, ks MetaClientKeystore, c
 	}
```

Comment:
Is the MetaClient currently active -- it looks Atlas-specific? 
I noticed a potential dependency in the metrics setup. If beholder is unavailable, will this cause the client to error out as well?

Ideally, we want to ensure the client is decoupled so that a beholder outage doesn't block the main flow. For example, on 10/10 I saw many alerts about beholder potentially having issues.

Reply:
The errors you saw are due to Beholder package not properly exlcuding the bootstrap node from the metrics and sending repeated error messages. This is Beholder-specific.